authentication/urls.py

```diff
@@ -8,7 +8,6 @@
 urlpatterns = [
     path('register/', RegisterView.as_view(), name='register'),
     path('login/', LoginView.as_view(), name='login'),
-    path('token/', TokenObtainPairView.as_view(), name='token_obtain_pair'),
     path('token/refresh/', TokenRefreshView.as_view(), name='token_refresh'),
     path('verify/', SendVerificationEmailView.as_view(), name='verify_email'),
     path('recover/', SendRecoverPasswordEmailView.as_view(), name='recover_password'),
```

Comment:
This commit removes the 'profile/' URL endpoint from the authentication/urls.py file. This endpoint was previously used for updating a profile but has been deleted, please put it back.

Reply:
![image](https://github.com/RevelioStartup/revelio-be/assets/88031010/79288cd5-9199-405a-bdf1-66fc7412f7a0)
Still exist though ...